src/Mayaqua/Network.c

```diff
@@ -5841,7 +5880,9 @@ SSL_PIPE *NewSslPipe(bool server_mode, X *x, K *k, DH_CTX *dh)
 			SSL_CTX_set_ssl_version(ssl_ctx, SSLv23_client_method());
 		}
 
-		//SSL_CTX_set_verify(ssl_ctx, SSL_VERIFY_PEER, cb_test);
+		if (verify_peer) {  

```

Comment:
The bracket should be in a new line, to maintain consistency with the current code style.